test/integration_test/storkctl_migrationschedule_test.go

```diff
@@ -100,11 +102,11 @@ func createCustomSyncMigrationScheduleTest(t *testing.T) {
 		"suspend":                          "",
 		"start-applications":               "",
 	}
-	createMigrationScheduleTest(t, testrailID, cmdArgs, "custom-sync-migration-schedule.yaml", defaultNs)
+	createMigrationScheduleTest(t, testrailID, cmdArgs, "custom-sync-migration-schedule.yaml", defaultNs, "")
 }
 
 func createMigrationScheduleTest(t *testing.T, testrailID int, args map[string]string,
-	specFileName string, migrationScheduleNs string) {
+	specFileName string, migrationScheduleNs string, validateSchedulePolicyInterval string) {
```

Comment:
nit..Why did we chose the name of this variable as **validate**SchedulePolicyInterval? Why not just 
schedulePolicyInterval or inputSchedulePolicyInterval.

Reply:
Makes sense. When I first created the variable I was planning for it to be a boolean. Later changed it but did not update the variable name. Will address.